package/libgluonutil/src/libgluonutil.h

```diff
@@ -26,11 +26,14 @@
 
 #ifndef _LIBGLUON_LIBGLUON_H_
 #define _LIBGLUON_LIBGLUON_H_
+#include <arpa/inet.h>
+#include <stdbool.h>
```

Comment:
Add a few blank lines here (one blank line between include guards and include lines, two between the include lines and the other declarations).

Reply:
done.